packages/build-info/src/metrics.test.ts

```diff
@@ -10,6 +10,7 @@ describe('metrics', () => {
     test('returns an error when passed a string', async () => {
       const errorSpy = vi.spyOn(console, 'error')
       report('error happened', { client: mockClient })
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
       expect(errorSpy).toHaveBeenCalledOnce
```

Comment:
I believe the fix for these is to change `toHaveBeenCalledOnce -> toHaveBeenCalledOnce()`